homeassistant/components/youless/sensor.py

```diff
@@ -22,344 +25,302 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.device_registry import DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
-from homeassistant.helpers.typing import StateType
-from homeassistant.helpers.update_coordinator import (
-    CoordinatorEntity,
-    DataUpdateCoordinator,
-)
 
 from . import DOMAIN
+from .coordinator import YouLessCoordinator
+from .entity import YouLessEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class YouLessSensorEntityDescription(SensorEntityDescription):
+    """Describes a YouLess sensor entity."""
+
+    device_group: str
+    device_group_name: str
+    friendly_name: str
+    icon: str | None
```

Comment:
let's remove the icon for now and lets add that in a followup, you can then just migrate the integration to use translations and add icon translations with that